### PR TITLE
Add Tricentis icon

### LINKED ISSUE #9156: Tricentis
### Brand Name

Tricentis

### Website

https://www.tricentis.com/

### Popularity Metric

https://www.similarweb.com/website/tricentis.com/#overview

### Official Resources for Icon and Color

https://github.com/Tricentis

### Additional Comments

{
   "title":"Tricentis",
   "hex":"12438C",
   "source":"https://www.tricentis.com"
}

### Contributing

Yes

### PR DESCRIPTION
**Pull request by abhinavminhas**
**Issue:** closes #9156

**Similarweb rank:** 109,500
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
- Took logo from Tricentis footer logo https://Tricentis.com and created a vector image manually in Inkscape.
- Used [SVGOMG Online Tool](https://jakearchibald.github.io/svgomg/) to refine svg path.
- Picked Tricentis branding color using a Inkscape pencil tool on https://www.tricentis.com/ logo

### Preview
tricentis_icon_preview

### CODE REVIEW

**Comment by abhinavminhas** [comment]
@service-paradis How do I make the path one dimensional without distorting the icon ?

x icon-size Size of <path> must be exactly 24 in one dimension; the size is currently 19.825 x 23.856

File: icons/tricentis.svg
Comment (adamrusted) [inline icons/tricentis.svg:1]: Spotting [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/4fe7ed66-963f-46aa-8b2d-73d55289d20f) between the SVG available in the footer, and this version. Could you take another look please @abhinavminhas 🙏

Comment (abhinavminhas) [reply icons/tricentis.svg:1]: @adamrusted thank for the review earlier. Can you please have a look at the new logo (taken from the Tridents footer logo). Update the preview above.

**Comment by abhinavminhas** [comment]
Do you mind reviewing @adamrusted ?

**Review by adamrusted — APPROVED** [review]

**Pull request merged**